### ISSUE TITLE
Restored Byron wallet -> no transaction possible

### ISSUE THREAD

**Issue by Octalus** [input-output-hk/daedalus#3228]
After restoring an old Byron wallet via seret.key file, deadalus shows the correct balance and displays transactions correctly.

But the newest version 6.0.1 is unable to create a correct transaction for this kind of wallet. "An Error occured" and on the node side no transaction gets submitted. The following error occurs:

`TxValidationErrorInMode (ShelleyTxValidationError ShelleyBasedEraAlonzo (ApplyTxError [UtxowFailure (WrappedShelleyEraFailure (MissingVKeyWitnessesUTXOW)`

I also tried to connect "cardano-wallet" against the node. But the command 'cardano-wallet wallet list' hides byron wallets. Only shelley wallets are listed with IDs which are needed to interact with them further via cardano-wallet cli commands.

**Comment by Tonytech-wq** [comment]
@Octalus Thanks for posting the issue. 

Your query would be best dealt with by the support team.

Please see the link below to our dedicated support line:

Support : [IOHK support](https://cardano-zendesk.web.app/)

Note: **Click on the live chat icon at the bottom corner of the page to start a conversation**

**Comment by Octalus** [comment]
![transaction_error](https://github.com/user-attachments/assets/f842e897-2d7c-460b-ba33-1546f19e153c)

This is the error in the node.log. Somehow the transaction is not constructed correctly

**Comment by Tonytech-wq** [comment]
Have you able to resolved your issue @Octalus

**Comment by Octalus** [comment]
@Tonytech-wq 
your support page is a scam page and wants to have my private keys for help. I'm not stupid!

**Comment by HeptaSean** [comment]
The `secret.key` file simply doesn't contain the correct private key to spend from this wallet.

Over the years, they have tried to scam *a lot* of people with exactly this `secret.key` file that looks like it gives access to [DdzFFzCqrhtDG8CnukNLfmDUyixQNgn7a1sEMunMfVXuqCnq7x7KkwyCvK88VSe9UBWxVvWUzYUD2cXYdWLkqn1fQPFqRCgbfWrA1uBP](https://adastat.net/addresses/DdzFFzCqrhtDG8CnukNLfmDUyixQNgn7a1sEMunMfVXuqCnq7x7KkwyCvK88VSe9UBWxVvWUzYUD2cXYdWLkqn1fQPFqRCgbfWrA1uBP) and [DdzFFzCqrhsmiiQJ9QCqgRz6jisU688V2mSnJQ2R8Hwz2gS6ezHoawJkiE9VhkHNCHCjz2AZ4RJ9cPy2Raq9RkGYacfsm7H4owtfQvo2](https://adastat.net/addresses/DdzFFzCqrhsmiiQJ9QCqgRz6jisU688V2mSnJQ2R8Hwz2gS6ezHoawJkiE9VhkHNCHCjz2AZ4RJ9cPy2Raq9RkGYacfsm7H4owtfQvo2), but it doesn't.

Hope that you didn't pay something to whomever gave that file to you.

For what it's worth, here is the `MissingVKeyWitnessUTXOW` that I got when someone convinced me to try my luck with it:
![Image](https://github.com/user-attachments/assets/a1106f17-3d17-425d-8484-c344625b9908)
You can see the same `ffea5f…3d2232` key hash that is also missing a signature in your case.

**Comment by michalrus** [comment]
> Over the years, they have tried to scam _a lot_ of people with exactly this `secret.key` file

Hey, thanks for the insight! Could you say what the actual scam is?

Do regular (deceived) users pay scammers for a `secret.key` file that almost works, and then try to contact our support, to make it work? 🤔

**Comment by HeptaSean** [comment]
> Could you say what the actual scam is?
> Do regular (deceived) users pay scammers for a secret.key file that almost works,

Example from when I tried to help with exactly this file:
<img src="https://github.com/user-attachments/assets/9f4b3330-41d4-4a13-88ed-da9f3f165633" width=50% height=50%>

Here is another similar one on the forum:
https://forum.cardano.org/t/cardano-a-question-regarding-forum-moderation-and-decentralised-governance/145993/8

So, yeah, they try to get people to pay for the file.

> and then try to contact our support, to make it work? 🤔

Probably.

Also possible that the people asking did not yet pay for the file, but are just trying to “help” other people who contacted them (and that they hope to get a percentage when it works).

Hard to tell which people in the chain really consciously try to scam and which (who maybe paid others for the file, maybe not) still honestly believe that there has to be a solution.

**Comment by HeptaSean** [comment]
Part of why this *might* sometimes work (I have personally only met people who wasted time, not money on this) is that Daedalus takes quite a while (actually days for me) to restore these old `secret.key` wallets (because it has to scan the whole blockchain history, I suppose). And during that time, it looks like the file actually gives access to this wallet:
<img src="https://github.com/user-attachments/assets/54400e9d-7081-4779-a58a-a21ebb0327d2" width=50% height=50%>

So, they have some time where people may think: “Oh, this looks like a walk in the park. Yeah, I can pay them something for this.”

For what it's worth, the hex dump of the `secret.key` I got is:
```
$ hexdump secret.key
0000000 8184 4158 4d70 19ae 8472 68fa 481c 14fd
0000010 d22c 3462 77f7 14aa c30c 2141 fff5 6c7a
0000020 0914 3f0e 3902 450f d3b4 7fd1 368b e018
0000030 cfae 917c e2b8 132f aaf5 799d fb1b 1028
0000040 9e48 15f6 8185 8058 4a88 9be6 3db4 b5e3
0000050 1833 c755 b645 e7c0 db2f 0dc0 14eb 099d
0000060 2d66 fca0 3dbe 4daf c07d 777b c6d7 a6f5
0000070 a8d8 a603 4960 8ee9 d9cb 5db2 29fb 4973
0000080 f104 ec7c 76d6 c2d5 ee68 2c9f abe1 f76e
0000090 ac6a 7815 a5eb dff9 84d2 a238 828d a1b5
00000a0 0493 8441 3170 2fa6 d6bd f64f a652 ae85
00000b0 3b52 c6ea 9726 9e9c f332 8db9 57d7 ba83
00000c0 ccf4 d4e7 f411 8e6e 829f 8058 9233 05a8
00000d0 42cc 2884 b1ec 99fd 28c2 0d96 9016 5b70
00000e0 1d00 a984 52ef 2202 b5ef 9540 7ada da74
00000f0 79a6 1641 a3db 20ae dfad 1500 70f2 1507
0000100 0f75 5fe5 59d8 849d e0a9 7afe 9d2d d4a9
0000110 709d edf0 c818 d0ec d8be 4fac f1f3 5afd
0000120 6679 eac8 2e5a 5339 5fbf 2c63 6502 4802
0000130 9f5b 13c2 6cbd 7d34 1b1b a032 bdff 4445
0000140 0d1c 2902 9e6d e6f7 a022 7236 6058 3431
0000150 387c 317c 777c 6933 5552 2b35 7468 7554
0000160 7047 754e 7852 7946 6e6b 5134 5666 4268
0000170 4c45 3166 5a34 564c 334e 4f4c 5261 6349
0000180 7c3d 4e48 747a 3358 6d42 7363 7646 3276
0000190 7735 2b76 5735 6d54 6d30 3732 3963 6761
00001a0 6942 435a 4c57 6770 6556 4f35 3d55 80ff
00001b0
```

Looks like some kind of CBOR structure, but I haven't dived deeper than that.

Maybe, Daedalus could analyse the file and warn already at the beginning that private and public key do not match?

**Comment by michalrus** [comment]
Okay, I see now! Thanks for the explanation :)

**Comment by HeptaSean** [comment]
On the forum ([https://forum.cardano.org/t/please-help-get-ada-from-old-wallet-lost-12-phrase-key/58390/9](https://forum.cardano.org/t/please-help-get-ada-from-old-wallet-lost-12-phrase-key/58390/9)), you said that this is *not* the wallet you are trying to restore.

Anyway, yes, over the years a *lot* of people were asked for help with exactly this wallet and then asked to pay with a discount for it. It is 99% a scam.

By the way, my sympathy is limited by the fact that you first claimed that this is *your* wallet in [https://github.com/cardano-foundation/cardano-wallet/issues/2395#issuecomment-2904263646](https://github.com/cardano-foundation/cardano-wallet/issues/2395#issuecomment-2904263646) as well as [https://forum.cardano.org/t/please-help-get-ada-from-old-wallet-lost-12-phrase-key/58390/7](https://forum.cardano.org/t/please-help-get-ada-from-old-wallet-lost-12-phrase-key/58390/7) and that you decided to do a baseless “Do I need to report daedalus company to FBI?” threat in the latter as well as that you seem to try to make a business model out of trying to access wallets on several chains that more proficient people have tried to access for years as shown by [https://github.com/DevKing1112/XRP](https://github.com/DevKing1112/XRP).

**Comment by HeptaSean** [comment]
Probably.

Chances are very slim that that amount of assets lie around on a chain *for years* without some people who know *everything* about that chain trying to help get them back.

*Selling* keys to such wallets is a huge red flag in any case.

**Comment by DevKing1112** [comment]
Sorry,
ADA and XRP are real.
Also, the HEX code you revealed is fake.
I am currently obtaining the information of the scammed users from ADA company and Ripple company.
A trial is needed.
I am sorry, how did you get this information?
Currently, the FBI is investigating at the request of the Hyogo Court in Japan.
Can you reveal your identity?
I need your cooperation.
Thanks!

**Comment by DevKing1112** [comment]
bb@bbraatz.eu](mailto:bb@bbraatz.eu

**Comment by DevKing1112** [comment]
Your mail address is right?

**Comment by HeptaSean** [comment]
> I am sorry, how did you get this information?

I was contacted by someone with a `secret.key` file for this wallet back in December 2024.

Did not pay them anything for it. If I help people with Cardano problems, I help them with the problem not with my money and they can pay whatever they like if (and only if) it is successful.

Only after Daedalus did a lot of syncing for days, I realised that that key wouldn't give access and remembered that we were warned in a Cardano moderator Telegram group about exactly this wallet already in 2022 (warning is quoted in [https://forum.cardano.org/t/cardano-a-question-regarding-forum-moderation-and-decentralised-governance/145993/7](https://forum.cardano.org/t/cardano-a-question-regarding-forum-moderation-and-decentralised-governance/145993/7)).

Now, I am seeing all these threads pop up on GitHub and on the forum with other people trying to open the exact same wallet. That's all.

> Your mail address is right?

Yes, information on this GitHub account is correct.

**Comment by DevKing1112** [comment]
Sorry,
Is the client from Japan?
I need his information.

We need to catch the scammer.
Please help us.

**Comment by HeptaSean** [comment]
> Sorry, Is the client from Japan? I need his information.

I don't know where the person who contacted me is from. He used a Telegram account with the display name “TK” and the username `@uiofgh2` (which interestingly is still active).

One of the other people contacted (who also did not pay anything for the file) said that their scammer used the name “Shoto Tanake”, a Proton mail account, and seemingly had the IP address `49.109.141.87` which is from a Japanese Internet provider and doesn't seem to be used by a VPN or Tor exit ([https://forum.cardano.org/t/cardano-a-question-regarding-forum-moderation-and-decentralised-governance/145993/10](https://forum.cardano.org/t/cardano-a-question-regarding-forum-moderation-and-decentralised-governance/145993/10)). I guess the IP comes from the mail headers, which theoretically could be forged.

Doesn't necessarily have to be always the same person, though.

**Comment by disassembler** [comment]
Closing as part of issue backlog cleanup — issues older than 90 days are being retired to give the new team a clean slate. Please open a new issue if this is still relevant.
